index.js

```diff
@@ -1,18 +1,38 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector('#header')
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
-
+header.style.color = "aqua"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
-
+console.log('This is what the traveler object looks like: ', traveler)
+const profileDiv = document.querySelector('#profile')
+profileDiv.innerHTML = `
+  <img src=${traveler.photo} alt=${traveler.name}>
+  <h2>${traveler.name}</h2>
+  <p>AKA: <em>${traveler.nickname}</em></p>
+`;
 
 /***** Deliverable 4 *****/
-
+const animalsList = document.querySelector("#animals")
+
+traveler.animalSightings.forEach(sighting => {
```

Comment:
good good good! 

- this isn't something we've really covered yet, but what if in the future, we were adding to our array of travelers? how could we reuse this logic to handle updates to our array? (hint hint...helper functionsss)